Azure/azure-cli
# [RDBMS] Improvements to Management Plane for MySQL and PostgreSQL

**Description**  

This PR improves existing commands for the 'Flexible Servers' APIs within the RDBMS component.  

Changes include:

- Update MySQL Connection string in create command
- Replace server name check with CheckNameAvailability API for MySQL & PostgreSQL
- Update help text and parameters for commands to be more descriptive and helpful to user (commands/arguments: vnet related, replica, firewall, restore).  These were to improve customer experience.
- Update validators to use result from list-skus commands/API.  
- Add command to enable user to connect to Flexible Server 
- Enabled more tests for code coverage and command testing (validator & replica tests)
- Added new recording files with default locations for MySQL & PostgreSQL 

**Testing Guide**  


To run tests (these cover existing commands):
azdev test rdbms

New commands: 
az mysql flexible-server connect --admin-user testuser --admin-passoword testpassword --name servername
---

This checklist is used to make sure that common guidelines for a pull request are followed.
